lib/security-release/security-release.js

```diff
@@ -165,16 +165,24 @@ export async function pickReport(report, { cli, req }) {
     defaultAnswer: await getSupportedVersions()
   });
 
-  let patchAuthors = await cli.prompt(
-    'Add github username of the authors of the patch (split by comma if multiple)', {
-      questionType: 'input',
-      defaultAnswer: ''
-    });
-
-  if (!patchAuthors) {
-    patchAuthors = [];
+  let prURL = '';
+  let patchAuthors = [];
+  if (custom_field_values.data.length) {
```

Comment:
If the custom attribute is not set it will throw, we might need to check if  present
```suggestion
  if (custom_field_values?.data.length) {
```

Reply:
The field `custom_field_values` will always exist, the difference is if it's empty (no custom fields). I have tested it with a couple of different reports (with and without the custom field).